core/resources/shaders/sdf.fs

```diff
@@ -68,16 +69,20 @@ float sampleAlpha(in vec2 uv, float distance, float threshold) {
 
 void main(void) {
 
-    float threshold_fill = 0.5;
-    float threshold_stroke = threshold_fill - v_strokeWidth * u_device_pixel_ratio;
-
     float distance = texture2D(u_tex, v_texcoords).a;
-
-    float alpha_fill = pow(sampleAlpha(v_texcoords, distance, threshold_fill), 0.4545);
-    float alpha_stroke = pow(sampleAlpha(v_texcoords, distance, threshold_stroke), 0.4545);
-
-    vec4 color = mix(v_strokeColor, v_color, alpha_fill);
-    color.a = max(alpha_fill, alpha_stroke) * v_alpha * v_color.a;
+    vec4 color;
+
+    if (u_pass == 1) {
```

Comment:
I know we're not filling a lot of fragments with SDF shaders, but do we know the trade-off in a branching shader vs. compiling a separate shader for each pass?